CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# How to contribute
+
+Add whatever it is that you would like to the README.md. You can fork the repo and edit locally, or simply use Github's built-in editor. 
+
+Try to follow existing formatting. In a lot of cases, it makes sense to alphabetize the items. If a list is already in alphabetical order, please add new items in alphabetical order as well.
```

Comment:
How about something like a opening paragraph to the tune of:

"This page was designed for the development industry of Phoenix.  It's primary goal is to help with jobs, furthering education, and betterment of the industry."